### ISSUE TITLE
[Warrior][Spell][charge/intercept]

### ISSUE THREAD

**Issue by ripsnarl** [dalaranwow/dalaran-wow#4741]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Charge/intercept currently is set to go to the center of a hit box of a player. When you charge you always will go to the center and thereby you often end up "jumping" at the end of a charge.

**Expected behaviour**: Tell us what should happen instead
Charge should go straight to the max melee range of a target, not the center.

**Steps to reproduce the problem**:

1. charge an enemy

**Include proofs for this behavior**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
https://www.youtube.com/watch?v=wmAGsDHR8j8 @ 3:44 (this is a retail video)
https://www.youtube.com/watch?v=zEgaACiI54U&feature=youtu.be (this is on dalaWoW)

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Intercept id: 20252
Charge id: 11578

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by plznoz** [comment]
The jumping part is to avoid falling through the ground when you charge uphill.
It's a hackfix (shitty fix), and has been like that for a good while.

I would also like to add that Intercept and charge should follow the target if it moves (doesn't do that currently).

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/3655, 

The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
